test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/DefaultActionModelBuilderTest.cs

```diff
@@ -321,6 +322,27 @@ public void GetActions_ConventionallyRoutedAction_WithHttpConstraints()
         }
 
         [Fact]
+        public void GetActions_AuthorizeFiltersAreCombinedWhenOverriden()
```

Comment:
Poor test name, the intent was that both base and derived authorize filters show up, as opposed to getting only one.  I'll rename it to BaseAuthorizeFiltersAreStillValidWhenOverriden